docs/hardware/devices/index.mdx

```diff
@@ -245,6 +245,19 @@ nRF52840-based development boards with GPS, and multiple ports to attach sensors
 | [Seeed Wio-WM1110 Dev Kit](./seeed-studio/wm1110?wm1110=wio-sdk-wm1110)   | nRF52840 | LR1110 | YES  | 5.3 | YES |
 | [Seeed Wio Tracker 1110](./seeed-studio/wm1110?wm1110=wio-tracker-wm1110) | nRF52840 | LR1110 | YES  | 5.3 | YES |
 
+
+## [PSMG smart glasses by Seeed] (coming soon)
+
+nRF52840-based development boards, and multiple ports to attach sensors via seeed grove connectors.
+
+
+| Name                                                                      | MCU      | Radio  | WiFi | BT  | GPS      |
+| :------------------------------------------------------------------------ | :------- | :----- | :--: | :-: | :-:      |
+| [Seeed esp32-sx1262-kit](./seeed-studio/wm1110?wm1110=wio-sdk-wm1110)     | nRF52840 | LR1110 | YES  | 5.3 | OPTIONAL |
```

Comment:
I don't think we'd want to merge this until the sub page is created, and you'll need to update this table